src/_includes/footer.html

```diff
@@ -61,7 +61,7 @@
         }, config.scriptTimeout);
         var d = false;
         var tk = document.createElement('script');
-        tk.src = '//use.typekit.net/' + config.kitId + '.js';
+        tk.src = 'https://use.typekit.net/' + config.kitId + '.js';
```

Comment:
Oh fun! I'd still think about doing it in a separate commit to explain the _why_ though!